gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
```

Comment:
> if I want to know if it is on or off, how do?

`is_active` will tell you if any LEDs are lit https://github.com/RPi-Distro/python-gpiozero/blob/master/gpiozero/devices.py#L357
(unfortunately that's not in the [docs](http://gpiozero.readthedocs.io/en/latest/api_boards.html#compositedevice) yet)

> A custom `__repr__` is a good idea

What do you suggest? Is the `__repr__` provided by [CompositeDevice](https://github.com/RPi-Distro/python-gpiozero/blob/master/gpiozero/devices.py#L315) not good enough?